apps/src/dance/DanceAiClient.ts

```diff
@@ -0,0 +1,96 @@
+import UntypedCachedPalettes from '@cdo/static/dance/ai/model/cachedpalettemap.json';
```

Comment:
nit: we could make these filenames a bit more readable, e.g. `cached-palette-map.json`.  (We seem to be rather inconsistent in `static/` but kebab case seems relatively common.)